fdbserver/TenantBalancer.actor.cpp

```diff
@@ -191,6 +193,7 @@ struct TenantBalancer {
 	SourceMovementRecord getOutgoingMovement(Key prefix) const {
 		auto itr = outgoingMovements.find(prefix);
 		if (itr == outgoingMovements.end()) {
+			TraceEvent(SevError, "TenantBalancerGetOutgoingMovementError").detail("MissPrefix", prefix.toString());
```

Comment:
This event is not quite so severe as `SevError` which indicates a problem that could terminate a process, etc.

```suggestion
			TraceEvent(SevWarn, "TenantBalancerGetOutgoingMovementError", tbi.id()).detail("Prefix", prefix);
```